files/base.rb

```diff
@@ -63,6 +63,24 @@ def team_data(lookup_key = nil)
     data
   end
 
+  def check_data(lookup_key = nil)
```

Comment:
I don't understand these functions. Are they just over-engineered ruby has lookups? Why do we need these?